app/views/course/assessment/submission/submissions/_submissions.html.slim

```diff
@@ -0,0 +1,12 @@
+table.table.submissions
+  thead
+    tr
+      th = t('.student_name')
+      th = t('.status')
+      th = t('.grade')
+      th = t('.experience_points')
+      th
+  tbody
+    - course_users.each do |course_user|
```

Comment:
The issue is `course_users` without submission will not be captured in the `@submissions` variable. That's why I need to get `@course_users` first >_<

Unless I make a wrong assumption? @allenwq